packages/app/client/src/ui/pages/Account/components/Dashboard.js

```diff
@@ -0,0 +1,222 @@
+import React from 'react'
+import Grid from '@material-ui/core/Grid'
+import BorderedInfo from './BorderedInfo'
+import Typography from '@material-ui/core/Typography'
+import { Button } from '../../../atoms'
+import { Badge, Edit, Like } from '../../../assets'
+import { makeStyles } from '@material-ui/core/styles'
+import { StructuredCard } from '../../../molecules'
+
+const useStyles = makeStyles(() => ({
+  buttons: {
+    marginTop: '10%',
+    width: '184px'
+  },
+  bordered: {
+    width: '185px',
+    height: '150px',
+    border: '2.2px solid #15B54A',
+    borderRadius: '10px'
+  },
+  topText: {
+    height: '56px',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center'
+  },
+  count: {
+    height: '87px',
+    padding: '8%',
+    fontSize: '45px',
+    color: '#15B54A'
+  },
+  header: {
+    height: '100%',
+    alignItems: 'center',
+    display: 'flex',
+    justifyContent: 'space-between'
+  },
+  seeAllButton: {
+    width: '85px',
+    height: '16px',
+    color: '#15B54A',
+    backgroundColor: 'white'
+  },
+  line: {
+    borderBottom: '1px solid #E7E7E7'
+  },
+  boxes: {
+    margin: '5%',
+    display: 'flex',
+    justifyContent: 'space-between'
+  },
+  editButton: {
+    marginRight: '5%',
+    width: '107px',
+    height: '42px'
+  },
+  viewProject: {
+    width: '139px',
+    height: '42px'
+  }
+}))
+
+function Dashboard(props) {
+  const { handleChange } = props
+  const classes = useStyles()
+  const compareByDate = (a, b) => {
+    if (a.date > b.date) return -1
+    if (a.date < b.date) return 1
+    return 0
+  }
+  return (
+    <>
+      <Typography style={{ marginBottom: '2%', fontSize: '34px' }} variant="h3">
+        Hello, {props.name}
+      </Typography>
+      <Grid item container xs={12}>
+        <Grid item xs={12} sm={12} md={6}>
+          <BorderedInfo size="normal">
+            <Typography style={{ paddingTop: '4%' }} variant="subtitle1">MY STATS</Typography>
+            <div className={classes.boxes}>
+              <div style={{ textAlign: 'center' }}>
+                <div className={classes.bordered}>
+                  <div className={classes.topText}>
+                    <img style={{ marginRight: '5%' }} src={Like} />
```

Comment:
You could use the LikeButton in the atoms folder if it is different from your goal please feel free do improve it